### PR TITLE
IBX-3783: Updated GET product GET vat

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-3783
| Versions      | master, 4.2

### CODE REVIEW

File: docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/code/GET/ProductGet.json.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "ProductGet": {
+        "languages": [
+            "eng-GB"
+        ]
+    }
+}
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/code/GET/ProductGet.json.example:7]:
```suggestion
}

```

File: docs/api/rest_api/rest_api_reference/input/examples/product/catalog/vat/region/GET/VatCategoryList.json.example [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            },
+            {
+                "_media-type": "application/vnd.ibexa.api.VatCategory+json",
+                "identifier": "none",
+                "region": "region_1",
+                "vatValue": null
+            }
+        ]
+    }
+}
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/examples/product/catalog/vat/region/GET/VatCategoryList.json.example:25]:
```suggestion
}

```

File: docs/api/rest_api/rest_api_reference/input/examples/product/catalog/vat/region/identifier/GET/VatCategory.json.example [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "VatCategory": {
+        "_media-type": "application/vnd.ibexa.api.VatCategory+json",
+        "identifier": "standard",
+        "region": "region_1",
+        "vatValue": 18
+    }
+}
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/examples/product/catalog/vat/region/identifier/GET/VatCategory.json.example:8]:
```suggestion
}

```

File: docs/api/rest_api/rest_api_reference/input/ez-product-vat.raml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        responses:
+            200:
+                body:
+                    application/vnd.ibexa.api.VatCategoryList+json:
+                        type: VatCategoryList
+                        example: !include examples/product/catalog/vat/region/GET/VatCategoryList.json.example
+            401:
+                description: Error - the user is not authorized to read this object.
+            404:
+                description: Error - the vat category region not exists.
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/ez-product-vat.raml:19]:
```suggestion
                description: Error - the vat category region does not exist.
```

File: docs/api/rest_api/rest_api_reference/input/ez-product-vat.raml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    /{identifier}:
+        get:
+            displayName: Load vat specific region
+            description: Loads a vat category for the given identifier.
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/ez-product-vat.raml:23]: And in all the other occurrences.
```suggestion
            description: Loads a VAT category for the given identifier.
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 6]
```diff
@@ -2430,3 +2430,10 @@ TaxonomyEntryAssignToContent:
 TaxonomyEntryUnassignFromContent:
     type: object
     description: ''
+
+VatCategoryList:
+    type: object
+    description: ''
+VatCategory:
+    type: object
+    description: ''
```
Comment (konradoboza) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:2439]:
```suggestion
    description: ''

```

**Review by konradoboza — CHANGES_REQUESTED** [review]

**Comment by konradoboza** [comment]
https://github.com/ezsystems/developer-documentation/pull/1752#discussion_r969479827 this hasn't been resolved, please change `vat` -> `VAT` whenever it appears.

**Review by konradoboza — APPROVED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**